NitroxModel/Packets/Packet.cs

```diff
@@ -1,59 +1,60 @@
 ﻿using System;
 using System.Collections;
 using System.Collections.Generic;
-using System.IO;
 using System.Linq;
 using System.Reflection;
-using System.Runtime.Serialization;
-using System.Runtime.Serialization.Formatters.Binary;
 using System.Text;
-using LZ4;
-using NitroxModel.DataStructures.Surrogates;
 using NitroxModel.Networking;
+using BinaryConverter = BinaryPack.BinaryConverter;
 
 namespace NitroxModel.Packets
 {
     [Serializable]
     public abstract class Packet
     {
-        private static readonly SurrogateSelector surrogateSelector;
-        private static readonly StreamingContext streamingContext;
-        private static readonly BinaryFormatter serializer;
-
-        private static readonly string[] blacklistedAssemblies = { "NLog" };
-
         private static readonly Dictionary<Type, PropertyInfo[]> cachedPropertiesByType = new();
         private static readonly StringBuilder toStringBuilder = new();
 
+        public static void InitSerializer()
+        {
+            // This will initialize the processor for Wrapper which will initialize all the others
+            _ = BinaryConverter.Serialize(new Wrapper());
```

Comment:
There is no need to create a new Wrapper here, Serialize\<Wrapper\>(default) will work.